garrysmod/gamemodes/sandbox/entities/entities/gmod_thruster.lua

```diff
@@ -364,115 +358,189 @@ end
 	Register the effects
 -----------------------------------------------------------]]
 
-list.Set( "ThrusterEffects", "#thrustereffect.none", { thruster_effect = "none" } )
+list.Set( "ThrusterEffects", "none", { print = "#thrustereffect.none" } )

```

Comment:
You cannot retroactively change these definitions like that. Addons rely on the expected format to stay the same.